pkg/controller/volume/persistentvolume/testing/testing.go

```diff
@@ -193,7 +194,7 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 				return true, obj, ErrVersionConflict
 			}
 			if reflect.DeepEqual(storedClaim, claim) {
-				klog.V(4).Infof("nothing updated claim %s", claim.Name)
+				logger.V(4).Info("Nothing updated claim", "claimName", claim.Name)
```

Comment:
"claimName" -> "PVC". Please check the entire PR for this.

Reply:
yeah. I have updated , thanks.